assets/src/components/routeLadder.tsx

```diff
@@ -25,52 +26,49 @@ const Header = ({ route }: { route: Route }) => {
   )
 }
 
-const Vehicle = ({ vehicle }: { vehicle: Vehicle }) => {
-  const dispatch = useContext(DispatchContext)
-
-  return (
-    <ul
-      className="m-route-ladder__vehicle"
-      style={{ border: "1px solid black" }}
-      onClick={() => dispatch(selectVehicle(vehicle.id))}
-    >
-      <li>
-        <em>id:</em> {vehicle.id}
-      </li>
-      <li>
-        <em>label:</em> {vehicle.label}
-      </li>
-      <li>
-        <em>timestamp:</em> {vehicle.timestamp}
-      </li>
-      <li>
-        <em>direction_id:</em> {vehicle.direction_id}
-      </li>
-      <li>
-        <em>route_id:</em> {vehicle.route_id}
-      </li>
-      <li>
-        <em>trip_id:</em> {vehicle.trip_id}
-      </li>
-      <li>
-        <em>stop status:</em> {vehicle.stop_status.status},<br />
-        <em>stop:</em> {vehicle.stop_status.stop_id}
-      </li>
-      <li>
-        <em>timepoint status:</em>{" "}
-        {vehicle.timepoint_status && vehicle.timepoint_status.status},<br />
-        <em>timepoint:</em>{" "}
-        {vehicle.timepoint_status && vehicle.timepoint_status.timepoint_id},
-        <br />
-        <em>fraction before:</em>{" "}
-        {vehicle.timepoint_status &&
-          vehicle.timepoint_status.fraction_until_timepoint}
-      </li>
-    </ul>
-  )
-}
+const Vehicle = ({ vehicle }: { vehicle: Vehicle }) => (
+  <ul className="m-route-ladder__vehicle" style={{ border: "1px solid black" }}>
```

Comment:
Removing dispatch here will cause merge conflicts with #45 and make it hard/impossible to view overlapped vehicles and vehicles without timepoints (which we're likely to want to view during development). Can we leave it in for now?

Reply:
Sure